content/docs/deployment/csminstallationwizard/src/templates/operator/csm-isilon-1.12.0.template

```diff
@@ -0,0 +1,484 @@
+#
```

Comment:
Can the CSM 1.8 templates be deleted now?

Reply:
Will check with the component owners and update. Thanks @santhoshatdell